deployment/docker-compose.test.yml

```diff
@@ -19,9 +19,8 @@ volumes:
 
```

Comment:
Please revert this @michelleln 
I think it is because of autoformat from your side